databaseconnect.py

```diff
@@ -116,7 +121,7 @@ def get_question_response(subject, root, verb):
                 found = 1
                 break
         if found == 1:
-            cur.execute(f"SELECT sentence FROM statement_table WHERE verb='{verb}'")
+            cur.execute(f"SELECT sentence FROM statement_table WHERE verb={verb}")
```

Comment:
Although select queries aren't at risk of data manipulation, it could return sensitive/incorrect data or can be used to crash the system by running a query that retrieves large amounts of data. I'd recommend sticking to the same practice as implemented in the `INSERT` queries.

Reply:
Hi @janakrajchadha ! Is this the way you are asking me to convert them into?
`cur.execute("SELECT sentence FROM statement_table WHERE verb= %s", str(verb))`.

Reply:
@Pihu1998 Yes, correct